showcase/ruby/Gemfile.lock

```diff
@@ -11,7 +11,7 @@ GEM
       googleauth (~> 0.6.2)
       grpc (>= 1.7.2, < 2.0)
       rly (~> 0.2.3)
-    google-protobuf (3.6.1-universal-darwin)
+    google-protobuf (3.7.1-universal-darwin)
```

Comment:
Pretty sure this is a legit version: https://rubygems.org/gems/google-protobuf/versions/3.7.1-universal-darwin